modules/database/src/admin/customEndpoints/customEndpoints.admin.ts

```diff
@@ -31,6 +31,88 @@ export class CustomEndpointsAdmin {
     private readonly customEndpointController: CustomEndpointController,
   ) {}
 
+  async exportCustomEndpoints(): Promise<UnparsedRouterResponse> {
+    return await this.database
+      .getSchemaModel('CustomEndpoints')
+      .model.findMany({})
+      .then(r =>
+        r.map(obj => {
+          delete obj._id;
+          delete obj.createdAt;
+          delete obj.updatedAt;
+          delete obj.__v;
+          return obj;
+        }),
+      );
+  }
+
+  async importCustomEndpoints(
+    call: ParsedRouterRequest,
```

Comment:
`schemaId` entries should be updated as they won't match the exported ones

Reply:
It is updated in line 75